pkg/action/cobra.go

```diff
@@ -36,6 +36,7 @@ func CobraImpl(a *Action, streams cli.Streams) (*cobra.Command, error) {
 					return err
 				}
 			}
+
```

Comment:
If there was no change or proper refactoring, we don't need an extra blank line.

Reply:
returned back